spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -248,15 +238,15 @@ public void addDataUnpartitionedAvro() {
             "CALL %s.system.add_files('%s', '`avro`.`%s`')",
             catalogName, tableName, fileTableDir.getAbsolutePath());
 
-    Assert.assertEquals(2L, result);
+    Assertions.assertThat(result).isEqualTo(2L);
```

Comment:
it's ok to statically import `assertThat()` in places where it's newly introduced